### ISSUE TITLE
Could you please specify the training strategy

### ISSUE THREAD

**Issue by Aaron9477** [uzh-rpg/rpg_public_dronet#28]
Hi guys!
What's a great job you did!
I try to change the structure of your model. I need retrain the network from the scratch, so I want to draw lesson from your training strategy. I have read your paper, but I haven't find how to get the finally model . Could you please specify the training strategy, like training epochs, when to change learning rate and so on.
Looking forward your reply!
Best wishes!
yours, Wang

**Comment by Aaron9477** [comment]
Hi guys!
What's more, why you use the grayscale image instead of RGB image? That's because transmitting grayscale is faster than RGB? Or the grayscale image is enough in this mission, because the model concentrate on the "line-like" patterns.
Looking forward your reply!
Best wishes!
yours, Wang

**Comment by antonilo** [comment]
Hi Wang,
I am happy you liked our work! As we say in the paper, we use hard-mining and the Adam optimizer to train our model. You can find all the details / hyper-parameters in the training code in this repository (that is the same we used). The reasons we used grayscale are exactly the ones you pointed out. Hope it helps!

**Comment by Aaron9477** [comment]
> Hi Wang,
> I am happy you liked our work! As we say in the paper, we use hard-mining and the Adam optimizer to train our model. You can find all the details / hyper-parameters in the training code in this repository (that is the same we used). The reasons we used grayscale are exactly the ones you pointed out. Hope it helps!

Hi antonilo,
Thanks so much for your reply! I have figured out the question by your help!
Best wishes!
yours, Wang
